### PR TITLE
CLI: Automate docker container creation using content view

### LINKED ISSUE #3047: CLI: Automate docker container creation using content view
Existing CLI Docker container tests are verifying docker container creation using docker hub. It would be nice to have at least 1 test checking if creation using content view, lifecycle environment, custom repo etc works OK.

[Here's a sample flow](https://gist.github.com/omaciel/42b67894e080664d1746#file-09-docker-sh-L34-L80)

### PR DESCRIPTION
**Pull request by abalakh**
Closes #3047 

Test result after commenting `skip_if_bug_open` and manual downgrade of `docker` package to valid version:
```
py.test tests/foreman/cli/test_docker.py -k test_create_container_content_view
======================================== test session starts =========================================
platform linux2 -- Python 2.7.5, pytest-2.8.2, py-1.4.30, pluggy-0.3.1
rootdir: /home/andriy/Downloads/robottelo, inifile: 
collected 51 items 

tests/foreman/cli/test_docker.py .

=================== 50 tests deselected by '-ktest_create_container_content_view' ====================
============================= 1 passed, 50 deselected in 344.08 seconds ==============================
```

### CODE REVIEW

File: tests/foreman/cli/test_docker.py [hunk 1]
```diff
@@ -1275,6 +1275,48 @@ def test_create_container(self):
...
+            with self.subTest(compute_resource['url']):
+                container = make_container({
+                    'compute-resource-id': compute_resource['id'],
+                    'organization-ids': [self.org['id']],
+                    'repository-name': repo['container-repository-name'],
+                    'tty': 'yes',
+                })
+                self.assertEqual(
+                    container['compute-resource'], compute_resource['name'])
+
```
Comment (oshtaier) [inline tests/foreman/cli/test_docker.py]: Maybe, we can add more assertions here to check whether proper repository assigned and etc. That will be applied only for that basic scenario, but in future ones

File: tests/foreman/cli/test_docker.py [hunk 2]
```diff
@@ -1275,6 +1275,48 @@ def test_create_container(self):
                     container['compute-resource'], compute_resource['name'])
 
     @run_only_on('sat')
+    @skip_if_bug_open('bugzilla', 1282431)
+    def test_create_container_content_view(self):
+        """@Test: Create docker container using content views, lifecycle
```
Comment (oshtaier) [inline tests/foreman/cli/test_docker.py]: Maybe
```"""@Test: Create docker container using custom content view, lifecycle environment and docker repository for local and external compute resources

```

**Comment by oshtaier** [comment]
ACK pending non-blocking comments

File: tests/foreman/cli/test_docker.py [hunk 3]
```diff
@@ -1275,6 +1275,48 @@ def test_create_container(self):
...
+        environments and custom repos for local and external compute resources
+
+        @Feature: Docker
+
+        @Assert: The docker container is created for each compute resource
+
+        @BZ: 1282431
+
```
Comment (elyezer) [inline tests/foreman/cli/test_docker.py]: This blank line is not required according PEP8.

**Comment by abalakh** [comment]
Updated & rebased
```
py.test tests/foreman/cli/test_docker.py -k test_create_container_content_view
======================================== test session starts =========================================
platform linux2 -- Python 2.7.5, pytest-2.8.2, py-1.4.30, pluggy-0.3.1
rootdir: /home/andriy/Downloads/robottelo, inifile: 
collected 51 items 

tests/foreman/cli/test_docker.py .

=================== 50 tests deselected by '-ktest_create_container_content_view' ====================
============================= 1 passed, 50 deselected in 403.23 seconds ==============================
```

**Comment by elyezer** [comment]
ACK

**Pull request merged**